libmscore/measure.cpp

```diff
@@ -1956,6 +1956,10 @@ void Measure::read(XmlReader& e, int staffIdx)
             else
                   qDebug("illegal measure size <%s>", qPrintable(e.attribute("len")));
             irregular = true;
+            if (_len.numerator() <= 0 || _len.denominator() <= 0) {
+                  e.raiseError(QObject::tr("MSCX error at line %1: invalid measure length: %2").arg(e.lineNumber()).arg(_len.toString()));
```

Comment:
Maybe be better "File format" instead of "MSCX"?

Reply:
I am not really sure, it is more like an error in data rather than file format itself. The "MSCX" option though doesn't fit that well too.